infra/tf/variables.tf

```diff
@@ -58,8 +58,6 @@ variable "droplet_size" {
 #   Misc    #
 #############
 
-# export variables in the format `TF_VAR_xyz` to expose them to terraform
-# TF_VAR_do_token=
```

Comment:
I actually found this comment to be helpful when setting up terraform locally

Reply:
I even have a better way coming with this pr using tfvars file

Reply:
nice!